docs/faq.md

```diff
@@ -83,7 +83,7 @@ For legacy systems, the target quality should be in line with the system's curre
 
 ### Where can I find more information about your metrics?
 
-Sigrid itself includes a brief explanation of all metrics in the "refactoring candidates" page. If you need more information, you can find the [Guidance for producers](https://www.softwareimprovementgroup.com/wp-content/uploads/2019/11/20190919-SIG-TUViT-Evaluation-Criteria-Trusted-Product-Maintainability.pdf) on our website, which provides more details on what is measured and why.
+Sigrid itself includes a brief explanation of all metrics in the "refactoring candidates" page. If you need more information, you can find the [Guidance for producers](https://www.softwareimprovementgroup.com/wp-content/uploads/SIG-TUViT-Evaluation-Criteria-Trusted-Product-Maintainability-Guidance-for-producers.pdf)) on our website, which provides more details on what is measured and why.
```

Comment:
I just noticed last minute actually looking at the resulting file that there was an extra parenthesis, that should be removed. With that it should be fine. 

```suggestion
Sigrid itself includes a brief explanation of all metrics in the "refactoring candidates" page. If you need more information, you can find the [Guidance for producers](https://www.softwareimprovementgroup.com/wp-content/uploads/SIG-TUViT-Evaluation-Criteria-Trusted-Product-Maintainability-Guidance-for-producers.pdf) on our website, which provides more details on what is measured and why.
```